src/NaturalSort/NaturalSort.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Buildium.NaturalSort</id>
+    <version>1.0.0</version>
+    <title>NaturalSortComparer</title>

```

Comment:
In this case, the Id of the project is `Buildium.NaturalSort`, but the name it would show up with in nuget.org would be `NaturalSortComparer`. That got me to thinking - how do we want to name our packages in Nuget? Using .NET namespacing with Buildium at the front, or with more functional names?